samples/python/bulk_edit_issues.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import json
+
+page_size = 1000 #max value
+page = 0
+api_key = "<api-key>"
+endpoint = "https://api.helpshift.com/v1/<domain>/issues"
```

Comment:
- Variables that are supposed to be provided by the user should be kept before the constants. It is also more readable if the sequence of variables is in order of importance. For eg, `api-key` and `endpoint` are more important, and without them, this script will not run.
- Constants can be in UPPER CASE.
- For domain, you can have another variable `domain`. This variable can then be used to construct `endpoint`.

Reply:
* I will fix the sequence.
* Yeah, I have modified this in the next patch.
* Yes, have done this too.